pages/index.js

```diff
@@ -55,7 +92,25 @@ export default function HomeController(props) {
     return newPlaylist;
   };
 
-  const loginFunction = async () => {
+  const loginFunction = async (props) => {
```

Comment:
You can actually get rid of everything except the fetch().then().then() chain here. The other stuff was from when I was thinking about having a separate page for logging in.